netcdf-sys/build.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    println!("cargo:rustc-link-lib=netcdf");
```

Comment:
it would be good to emit a rebuild-if-changed line. Otherwise it rebuilds on any file changing, including any loose text files.

Reply:
Would that mean emitting rebuild-if-changed for the three files used here?

Reply:
we only need to emit a dummy actually, because the output of the build script only depends on the build script itself. It's somewhat of an oddity, that the default is to watch any file. So emitting any file or just "build.rs" should be good.

Reply:
New PR in #22 fixes this